object/project.go

```diff
@@ -0,0 +1,54 @@
+package object
+
+import (
+	"time"
+	"xorm.io/builder"
+)
+
+type Project struct {
+	Id        int64
+	CreateAt  time.Time        `xorm:"created" json:"create_at"`
+	UpdatedAt time.Time        `xorm:"updated" json:"updated_at"`
+	DeletedAt time.Time        `xorm:"deleted" json:"deleted_at"`
+	Uuid      string           `xorm:"varchar(256) unique" json:"uuid"`
+	Creator   string           `xorm:"varchar(256) index" json:"creator"`
+	Status    int              `json:"status"`
+	BasicInfo ProjectBasicInfo `xorm:"mediumtext" json:"basic_info"`
+}
+
+type ProjectBasicInfo struct {
+	Name        string   `xorm:"index" json:"name"`
+	Description string   `json:"description"`
+	Requirement string   `json:"requirement"`
+	Target      string   `json:"target"`
+	GradeRange  []string `xorm:"mediumtext" json:"grade_range"`
+	Subjects    []string `xorm:"mediumtext" json:"subjects"`
+	Summary     string   `json:"summary"`
+}
+
+func GetProject(uuid string) *Project {
```

Comment:
Plz just copy the useful code from:

https://github.com/open-ct/openitem/blob/de5bfc4185fb368bfe947d10ae8f721a2c8ca8f6/object/dataset.go#L48-L108

Then rename the object name